rge_main.py

```diff
@@ -138,10 +126,13 @@ def eval_model(epoch: int) -> tuple[float, float]:
     args = get_params().parse_args()
     torch.manual_seed(args.seed)
 
-    device, train_loader, test_loader = preprocess(args)
-    model, criterion, optimizer, scheduler, grad_estimator = prepare_settings(
-        args, device
-    )
+    # set num_clients = 1 to make sure there's 1 train_loader
+    args.num_clients = 1
```

Comment:
what is this? and why?

Reply:
num_clients controls data splition, set it to 1 means there's no train data split. which is our use case in rge_main.

Wondering if we should delete this script so that we can focus on FL part